src/mapd-con-es6.js

```diff
@@ -227,7 +227,10 @@ export class MapdCon {
     this.buildTRasterPointTransformMap()
     this.buildTSourceTypeMap()
 
-    console.log(`this.TRasterPointTransformMap => `, this.TRasterPointTransformMap)
+    console.log(
+      `this.TRasterPointTransformMap => `,
+      this.TRasterPointTransformMap
+    )
     console.log(`this.TSourceTypeMap => `, this.TSourceTypeMap)
```

Comment:
@bmatcuk would you mind deleting these two console logs? They snuck into a previous PR.